build/yamls/antrea.yml

```diff
@@ -345,7 +347,11 @@ apiVersion: v1
 data:
   antrea-agent.conf: |
     # FeatureGates is a map of feature names to bools that enable or disable experimental features.
-    #featureGates:
+    featureGates:
+    # Enable antrea proxy which provides ServiceLB for in-cluster services in antrea agent.
+    # It should be enabled on Windows, otherwise NetworkPolicy will not take effect on
+    # Service traffic.
+    #  AntreaProxy: false
```

Comment:
I added features here but removed based on @ksamoray's comment https://github.com/vmware-tanzu/antrea/pull/847#issuecomment-645826682.
I think anyway we will need a doc to list all experimental features and their stage like:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features
would let other reviewers to comment whether they need to be listed here too.

Reply:
I feel ok to add relatively mature features to the yamls but not all features. Creating a separate doc sounds good.